rq/worker.py

```diff
@@ -233,6 +244,7 @@ def __init__(
         disable_default_exception_handler: bool = False,
         prepare_for_work: bool = True,
         serializer=None,
+        dequeue_strategy=DequeueStrategy.DEFAULT,
```

Comment:
What do you think of moving this argument to `work()`? `__init__` is starting to have too many kwargs to my liking.

Reply:
I don't particularly care about that many `kwargs`, since I feel it delivers a better DX with low marginal cost, but regardless of the `kwargs` count, I think it makes more sense to have this at `work()` indeed, a worker shouldn't really care about which strategy to use except when actually working

Reply:
👍

Reply:
While looking at this just noticed it was already on `work()`. The reason I had it on `__init__` was to make it flexible for the user to chose when/where to use. I'll delete anyway to reduce the number of  kwargs.